css/badge-card-1.css

```diff
@@ -0,0 +1,38 @@
+@import url(../../../../css/component.css);
+.card-badge-container {
+  background-color: var(--font-color);
+  position: relative;
+  height: 15rem;
+  width: 30rem;
+  margin-left: auto;
+  margin-right: auto;
+  box-shadow: 5px 5px 5px var(--shadow-color);
+  border-radius: 2rem;
+  color: var(--background-color);
+}
+
+.heading-2 {
```

Comment:
this could be card-heading-2 . naming consistency is very important if you are going to reuse these classes.. Card-name, card-heading... Just a suggestion